src/TimeGrid.jsx

```diff
@@ -74,6 +74,17 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
     this.applyScroll();
+
+    this.positionTimeIndicator();
+
+    // Update the position of the time indicator every minute
+    this._timeIndicatorInterval = window.setInterval(() => {
```

Comment:
better to use a recursive `setTimeout` here, setInterval can "drift" depending on how clogged the JS event loop is, which can cause it to update rather closely together vs actually every minute. `setTimeout` will ensure that its not called more than once every minute